src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { arrayOf, bool, node } from 'prop-types';
+import glamorous from 'glamorous';
+
+import Button from '../Button/Button';
+import constants from '../../constants';
+
+const { borderRadius } = constants;
+const { Div } = glamorous;
+
```

Comment:
You could include this in the import above. It would just be:
`import glamorous, { Div } from 'glamorous';`